py-polars/tests/unit/streaming/test_streaming_cse.py

```diff
@@ -48,12 +48,7 @@ def test_cse_expr_selection_streaming(monkeypatch: Any, capfd: Any) -> None:
     )
     assert_frame_equal(result, expected)
 
-    err = capfd.readouterr().err
-    assert "df -> projection[cse] -> ordered_sink" in err
-    assert "df -> hstack[cse] -> ordered_sink" in err
```

Comment:
We're no longer checking that cse is performed, because it's not reported in the executor any more.